docs/source/development/commands_reference.md

```diff
@@ -58,16 +58,16 @@ Here is a list of Kedro CLI commands, as a shortcut to the descriptions below. P
   * [`kedro new`](#create-a-new-kedro-project)
 
 * Project-specific Kedro commands
-  * [`kedro activate-nbstripout`](#strip-output-cells)
-  * [`kedro build-docs`](#build-the-project-documentation)
-  * [`kedro build-reqs`](#build-the-projects-dependency-tree)
+  * [`kedro activate-nbstripout (deprecated from version 0.19.0)`](#strip-output-cells)
```

Comment:
Fixed in the latest commits.

Reply:
I didn't mean to remove the "(deprecated from version 0.19.0)" completely, just to have it outside of the linked text so it would be: [`kedro activate-nbstripout`](#strip-output-cells)(deprecated from version 0.19.0) etc.

Reply:
Sure, I fixed this in the latest commit.